catalog/spatial/spatial-commands/src/main/java/org/codice/ddf/commands/spatial/geonames/GeoNamesUpdateCommand.java

```diff
@@ -20,6 +20,10 @@
 import org.apache.karaf.shell.api.action.Option;
 import org.apache.karaf.shell.api.action.lifecycle.Reference;
 import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.spatial.geocoding.FeatureExtractionException;
+import org.codice.ddf.spatial.geocoding.FeatureExtractor;
+import org.codice.ddf.spatial.geocoding.FeatureIndexer;
+import org.codice.ddf.spatial.geocoding.FeatureIndexingException;
```

Comment:
Since we extended this beyond geonames data, should we refactor it to be a GazetteerUpdateCommand and change the command to gazetteer.  And also update the description.

Reply:
(done)